revive.toml

```diff
@@ -0,0 +1,30 @@
+ignoreGeneratedHeader = false
```

Comment:
I think you meant to have this change in the next diff in the stack.

Reply:
fixed